styles/components/button.css

```diff
@@ -0,0 +1,77 @@
+.btn {
+    font-size: 1.6rem;
+    padding: 0.8rem 1.2rem;
+    margin: 0.5rem;
+    outline-style: none;
+    border-style: none;
+    border-radius: 0.5rem;
+    cursor: pointer;
+    letter-spacing: 0.08em;
+}
+
+.btn-logo {
+    width: 3rem;
+    height: 3rem;
+    cursor: pointer;
+}
+
+.btn-txt {
+    padding: 0.5rem;
+    color: var(--color-black);
+    cursor: pointer;
+}
+
+.btn-icon {
+    display: flex;
+    align-items: center;
+}
+
+.btn-txt:hover {
+    padding-bottom: 3px;
+    border-bottom: 2px solid var(--color-black);
+}
+
+.btn-primary-solid {
+    color: var(--color-white);
+    background-color: #0ea5e9;
+    border: 2px solid #0ea5e9;
+}
+
+.btn-primary-solid:hover {
+    background-color: #0d7caf;
+}
+
+.btn-primary-outline {
+    color: #0ea5e9;
```

Comment:
you can define variables for the hex colors and use them

Reply:
Yes sure thanks